components/ChallengeFilters/FiltersCardsType/FiltersCardsType.jsx

```diff
@@ -14,7 +14,7 @@ const FiltersCardsType = ({ isCardTypeSet }) => (
     <a
       href=""
       className={`challengs-tab ${isCardTypeSet === 'Challenges' ? 'active' : ''}`}
-      onClick={() => false}
+      onClick={(e) => e.preventDefault()}
```

Comment:
prevent link default behaviour, with React.js returning false is not enough
[https://facebook.github.io/react/docs/handling-events.html](https://facebook.github.io/react/docs/handling-events.html)